gptme/server/api.py

```diff
@@ -48,8 +48,12 @@ def api_conversations():
 @api.route("/api/conversations/<string:logfile>")
 def api_conversation(logfile: str):
```

Comment:
We may want to run this in a new process, see other comment.

(not in this PR)